2-mandatory/3-financial-times.js

```diff
@@ -23,59 +40,88 @@ function titleWithFewestWords(allArticleTitles) {
     (Hint: remember that you can also loop through the characters of a string if you need to)
 */
 function headlinesWithNumbers(allArticleTitles) {
-    // TODO
+  // TODO
+  let newHeadline = [];
+  for (let i = 0; i < allArticleTitles.length; i++) {
+    for (let letter of allArticleTitles[i]) {
+      if (
+        letter === "0" ||
```

Comment:
Your logic works, but this could be simplified, and easier to maintain by doing `letter >= '0' && letter <= '9'`

Reply:
This was amazing. Less code and more efficient.